app/uk/gov/hmrc/helptosave/models/HTSEvent.scala

```diff
@@ -154,3 +154,16 @@ object AccountCreated {
     implicit val format: Format[ManuallyEnteredDetails] = Json.format[ManuallyEnteredDetails]
   }
 }
+
+case class GetAccountResult(nino: String, account: JsValue)
+
+object GetAccountResult {
+  implicit val format: Format[GetAccountResult] = Json.format[GetAccountResult]
+}
+
+case class GetAccountResultEvent(getAccountResult: GetAccountResult, path: String)(implicit hc: HeaderCarrier, appConfig: AppConfig) extends HTSEvent {
+  val value: ExtendedDataEvent = {
```

Comment:
there's a `HtsEventSpec` you can add to to add a unit test to check that the fields are written as expected

Reply:
added unit test